docs/api/remix.md

```diff
@@ -476,7 +476,7 @@ function SomeComponent() {
   return (
     <button
       formAction={useFormAction("destroy")}
-      formMethod="DELETE"
+      formMethod="POST"
```

Comment:
[The docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Button#attr-formmethod) only talk about `get` or `post` (lowercase), so maybe we should use lowercase too?

```suggestion
      formMethod="post"
```